src/Session.php

```diff
@@ -480,6 +480,10 @@ public function getCacheLimiter()
      */
     public function setCookieParams(array $params)
     {
+        if ($this->isStarted()) {
+            return;
```

Comment:
Do you think we should throw an exception and raise an error than suppressing it

Reply:
I think so.

Reply:
@harikt Is the exception name something like `SessionAlreadyStartedExceptionton`?

Reply:
I think we should follow the Aura.Router convention. No Exception at end. Rest looks good to me.